### ISSUE TITLE
Error when updating to 1.8.0: Argument 1 passed to PhpOffice\PhpSpreadsheet\Reader\Xlsx\Styles::setStyleBaseData() must be an instance of PhpOffice\PhpSpreadsheet\Reader\Xlsx\Theme, null given

### ISSUE THREAD

**Issue by kissifrot** [PHPOffice/PhpSpreadsheet#1043]
This is:

```
- [ x] a bug report
```

### What is the expected behavior?

No error when loading a (valid) XLSX file

### What is the current behavior?

```
TypeError: Argument 1 passed to PhpOffice\PhpSpreadsheet\Reader\Xlsx\Styles::setStyleBaseData() must be an instance of PhpOffice\PhpSpreadsheet\Reader\Xlsx\Theme, null given, called in vendor/phpoffice/phpspreadsheet/src/PhpSpreadsheet/Reader/Xlsx.php on line 584
```

### What are the steps to reproduce?

```php
<?php

require __DIR__ . '/vendor/autoload.php';

use PhpOffice\PhpSpreadsheet\IOFactory;

$filepath = 'a/valid/xlsx/file';
$reader = IOFactory::createReaderForFile($filepath);
$reader->setReadDataOnly(true);
$spreadsheet = $reader->load($filepath);

// add code that show the issue here...
```
### Which versions of PhpSpreadsheet and PHP are affected?

PhpSpreadsheet v1.8.0
PHP 7.2.19

**Comment by vedmant** [comment]
I have the same issue after upgrading from 1.3.

**Comment by karser** [comment]
I encountered the same issue The error is only for xls**x**. Xls works fine.

**Comment by MarkBaker** [comment]
Should be resolved in 1.8.1
